app/views/projects/show.html.erb

```diff
@@ -64,7 +64,7 @@
     <% unless @project.parent_id %>
       <%= link_to 'Add Sub-Project', project_new_sub_project_path(@project.id), id:"subproject", class: "button magenta" %>
     <% end %>
-
+    <%= link_to "Archive Project", project_toggle_archive_path(@project.id), method: "patch", id: "archive", class: "button magenta", remote: true %>
```

Comment:
maybe this id can be `toggle_archive` instead of `archive` since it does both things?